metrics/network/iperf3_kubernetes/k8s-network-metrics-iperf3.sh

```diff
@@ -19,14 +19,15 @@
 #  container-server <----> host-client
 
 set -e
+set -x
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 
 source "${SCRIPT_PATH}/../../../.ci/lib.sh"
 source "${SCRIPT_PATH}/../../lib/common.bash"
 test_repo="${test_repo:-github.com/kata-containers/tests}"
 iperf_file=$(mktemp iperfresults.XXXXXXXXXX)
-TEST_NAME="${TEST_NAME:-IPerf}"
+TEST_NAME="${TEST_NAME:-network iperf3}"
```

Comment:
Nit: I'd be slighly wary about spaces in names as not everyone quotes every variable (unfortunately ;)